src/components/Header.jsx

```diff
@@ -7,8 +7,49 @@ const Header = () => {
   const [current, setCurrent] = useState("");
   const navigate = useNavigate();
 
+  const [isLoggedIn, setIsLoggedIn] = useState(false);
+  console.log("isLoggedIn", isLoggedIn);
+
+  const confirmLogin = async () => {
+    try {
+      // Include credentials to send cookies with the request
+      const response = await fetch("/auth/confirm", {
+        credentials: "include",
+        headers: {
+          "Cache-Control": "no-cache",
```

Comment:
When i was initially testing out the /confirm route, I would get a 304 'not modified' status code in response after /confirm was first checked due to caching. I wanted the actual 200 status code. This prevented that issue but i'm testing it out again, and am not sure if this was the solution or reason that everything started working. 

When i remove  `"Cache-Control": "no-cache",` now, I still get the 200 status code. So i may have been doing something wrong before

Reply:
Yea I'm actually not totally sure about this. I'll look into cache-control and get back to you here

Reply:
I still don't totally understand this but I don't think that its doing anything currently because my server doesn't do anything with the values. I could be wrong though. But lets remove it for now and see if its still working